jax/numpy/lax_numpy.py

```diff
@@ -504,6 +504,24 @@ def exp2(x):
   return lax.exp(lax.mul(lax.log(_constant_like(x, 2)), x))
 
 
+@_wraps(onp.ldexp)
+def ldexp(x1, x2):
+  x1, x2 = _promote_shapes("ldexp",
```

Comment:
Normally both of these functions are implemented by bit-tricks. It's possible the bit-trick implementations are faster. This implementation is however fine, it's just an observation.

Reply:
The bit trick is indeed the underlying implementation of C/C++, but I wonder how to access bit representation of floating point in JAX. Seems that there is currently no equivalent function of [np.ndarray.view](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.view.html) to reinterpret the data type.

Reply:
https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.bitcast_convert_type.html#jax.lax.bitcast_convert_type

would do the trick.

Reply:
Hi, I have implemented it with bitwise operations. Mainly based on golang's implementation.

https://github.com/golang/go/blob/master/src/math/ldexp.go
https://github.com/golang/go/blob/master/src/math/frexp.go